server/db/models/user.js

```diff
@@ -22,12 +17,30 @@ module.exports = (sequelize, DataTypes) => {
   }
   user.init(
     {
-      googleId: DataTypes.STRING,
-      displayName: DataTypes.STRING,
-      image: DataTypes.STRING,
-      email: DataTypes.STRING,
-      firstName: DataTypes.STRING,
-      lastName: DataTypes.STRING,
+      googleId: {
+        type: DataTypes.STRING,
+        allowNull: true,
+      },
+      displayName: {
+        type: DataTypes.STRING,
+        allowNull: true,
+      },
+      image: {
+        type: DataTypes.STRING,
+        allowNull: true,
+      },
+      email: {
+        type: DataTypes.STRING, //only required field. Allows invitations to projects and users can complete sign up later
```

Comment:
solid use of comment 👌

Reply:
though maybe could put email closer to the top of the schema so when you read the file you see it earlier hmm